product-mini/platforms/windows/CMakeLists.txt

```
@@ -154,3 +154,13 @@ target_link_libraries (libiwasm ${LLVM_AVAILABLE_LIBS} ${UV_A_LIBS})
 if (MINGW)
   target_link_libraries (libiwasm ws2_32)
 endif ()
+
+if (WIN32)
+  target_link_libraries(libiwasm ntdll)
+endif()
+
+if (WIN32)
+  target_link_libraries(iwasm ntdll)
+endif()
```

Could we merge those two into one?
```suggestion
if (WIN32)
  target_link_libraries(libiwasm ntdll)
  target_link_libraries(iwasm ntdll)
endif()
```